sdk/core/core-client-lro-rest/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
 
-## 1.0.0-beta.1 (2021-08-05)
+## 1.0.0-beta.1 (2021-08-06)
```

Comment:
I do not think we can release on a Friday, or do you have an exception from @ramya-rao-a?
```suggestion
## 1.0.0-beta.1 (2021-08-09)
```

Reply:
@ramya-rao-a can I have an exception? This is currently not really used in any published packages but will enable me to start working on the generator changes that will consume this package

Reply:
Spoke offline with Ramya she's okay releasing today